src/monster.cpp

```diff
@@ -641,9 +641,47 @@ static std::pair<std::string, nc_color> hp_description( int cur_hp, int max_hp )
         col = c_red;
     }
 
+    if( debug_mode ) {
+        damage_info += "  ";
+        damage_info += string_format( _( "%1$d/%2$d HP" ), cur_hp, max_hp );
+    }
+
     return std::make_pair( damage_info, col );
 }
 
+
+static std::pair<std::string, nc_color> speed_description( float mon_speed_rating,
+        bool immobile = false )
+{
+    if( immobile ) {
+        return std::make_pair( _( "It is immobile." ), c_green );
+    }
+
+    const std::array<std::tuple<float, nc_color, std::string>, 8> cases = {
+        std::make_tuple( 1.30f, c_red, _( "It is much faster than you." ) ),
+        std::make_tuple( 1.00f, c_yellow, _( "It is faster than you." ) ),
+        std::make_tuple( 0.70f, c_white_yellow, _( "It is a bit faster than you." ) ),
+        std::make_tuple( 0.55f, c_white, _( "It is about as fast as you." ) ),
+        std::make_tuple( 0.50f, c_white_cyan, _( "It is a bit slower than you." ) ),
+        std::make_tuple( 0.40f, c_cyan, _( "It is slower than you." ) ),
+        std::make_tuple( 0.20f, c_green, _( "It is much slower than you." ) ),
+        std::make_tuple( 0.00f, c_green, _( "It is practically immobile." ) )
+    };
```

Comment:
`clang++-3.8` check on Travis CI threw a warning here:
```suggestion
    const std::array<std::tuple<float, nc_color, std::string>, 8> cases = {{
        std::make_tuple( 1.30f, c_red, _( "It is much faster than you." ) ),
        std::make_tuple( 1.00f, c_yellow, _( "It is faster than you." ) ),
        std::make_tuple( 0.70f, c_white_yellow, _( "It is a bit faster than you." ) ),
        std::make_tuple( 0.55f, c_white, _( "It is about as fast as you." ) ),
        std::make_tuple( 0.50f, c_white_cyan, _( "It is a bit slower than you." ) ),
        std::make_tuple( 0.40f, c_cyan, _( "It is slower than you." ) ),
        std::make_tuple( 0.20f, c_green, _( "It is much slower than you." ) ),
        std::make_tuple( 0.00f, c_green, _( "It is practically immobile." ) )
    }};
```